recursion/tower_of_hanoi.go

```diff
@@ -0,0 +1,21 @@
+package recursion
```

Comment:
add a package description in a comment above package declaration

```suggestion
// Package recursion
package recursion
```